nautobot/extras/views.py

```diff
@@ -503,14 +505,40 @@ class DynamicGroupListView(generic.ObjectListView):
 class DynamicGroupView(generic.ObjectView):
     queryset = DynamicGroup.objects.all()
 
+    def get_memberships(self, objects):
+        return DynamicGroupMembership.objects.filter(group__in=objects)
+
     def get_extra_context(self, request, instance):
         context = super().get_extra_context(request, instance)
         model = instance.content_type.model_class()
         table_class = get_table_for_model(model)
+        dgm_table_class = get_table_for_model(DynamicGroupMembership)
 
         if table_class is not None:
-            members = instance.get_queryset()
-            members_table = table_class(members, orderable=False)
+            members_table = table_class(instance.members, orderable=False)
+
+            children = self.get_memberships(instance.children.all())
+            children_table = dgm_table_class(children, orderable=False)
+
+            descendants = instance.get_descendants()
+            descendants_table = NestedDynamicGroupDescendantsTable(
+                descendants,
+                orderable=False,
+                sequence=["name", "members", "description"],
+                exclude=["content_type"],
+            )
+            descendants_tree = instance.flatten_tree(instance.descendants_tree())
+            descendants_map = {node.name: node.depth for node in descendants_tree}
+
+            ancestors = instance.get_ancestors()
+            ancestors_table = NestedDynamicGroupAncestorsTable(
+                ancestors,
+                orderable=False,
+                sequence=["name", "members", "description"],
+                exclude=["content_type"],
+            )
```

Comment:
Both of these tables should now also be membership tables, topologically sorted.